src/ui/components/TransferGrain.js

```diff
@@ -18,90 +17,151 @@ export type Props = {|
-        width: "80%",
-        margin: "0 auto",
-        background: "white",
-        padding: "0 5em 5em",
-      }}
-    >
-      <h1>Transfer Grain</h1>
-      <form onSubmit={(e) => submitTransfer(e)}>
-        <label>From</label>
-        <AccountDropdown ledger={ledger} setCurrentIdentity={setSender} />
-        <br />
-        <label>To</label>
-        <AccountDropdown ledger={ledger} setCurrentIdentity={setReceiver} />
-        <p>
-          <label htmlFor="amount">Amount</label> <br />
-          <input
-            type="number"
-            name="amount"
-            min="0"
-            step="any"
-            placeholder={`max: ${maxAmount}`}
-            required
-            value={amount}
-            onChange={(e) => setAmount(e.currentTarget.value)}
+    <div className={classes.root}>
+      <div className={classes.centerRow}>
+        <h1>Transfer Grain</h1>
+      </div>
+      <div className={classes.centerRow}>
+        <div
+          className={`${classes.dropdown} ${classes.centerRow} ${classes.third}`}
+        >
+          <AccountDropdown
+            ledger={ledger}
+            setCurrentAccount={setSender}
+            placeholder="From..."
           />
-          <span>{sourceIdentity && ` max: ${format(maxAmount, 5)}`}</span>
-        </p>
-        <p>
-          <label htmlFor="memo">Memo</label> <br />
-          <input
-            type="text"
-            name="memo"
-            value={memo}
-            onChange={(e) => setMemo(e.currentTarget.value)}
+        </div>
+        <div className={`${classes.centerRow} ${classes.third}`}>
+          <div className={classes.arrowBody}>
+            <div style={{width: "100px", display: "inline-block"}}>
+              <Input
+                style={{color: "black"}}
+                disableUnderline
+                type="number"
+                min="0"
+                placeholder="(amount)"
+                value={amount !== "0" ? amount : ""}
+                onChange={(e) => setAmount(e.currentTarget.value)}
+              />
+            </div>
+            <span>{sender && ` max: ${format(sender.balance, 2)}`}</span>
+          </div>
+          <div className={classes.triangle} />
+        </div>
+        <div
+          className={`${classes.dropdown} ${classes.centerRow} ${classes.third}`}
+        >
+          <AccountDropdown
+            ledger={ledger}
+            setCurrentAccount={setReceiver}
+            placeholder="To..."
           />
-        </p>
-        <input
+        </div>
+      </div>
+      <div className={classes.centerRow}>
+        <Input
+          className={classes.memo}
+          disableUnderline
+          type="text"
+          value={memo}
+          placeholder="Memo"
+          onChange={(e) => setMemo(e.currentTarget.value)}
+        />
+      </div>
+      <div className={classes.centerRow}>
+        <Button
+          className={`${classes.centerRow} ${classes.half}`}
+          style={{background: "#6558F5"}}
```

Comment:
Shouldn't hardcode colours like this. Instead, use the color prop so it can pull values from the theme: 
```suggestion
          color="primary"
```